### PR TITLE
update gitextractor.md

### PR DESCRIPTION
**Pull request by shubham-cmyk**
Finding Repo ID could be time-consuming to figure out the repo id just run the query in the browser console.

### CODE REVIEW

**Review by mindlesscloud — APPROVED** [review]
LGTM

File: docs/Plugins/gitextractor.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ curl --location --request POST 'localhost:8080/pipelines' \
 ```
 - `url`: the location of the git repository. It should start with `http`/`https` for a remote git repository and with `/` for a local one.
 - `repoId`: column `id` of  `repos`.
+   Note : To find the repo id run `$("meta[name=octolytics-dimension-repository_id]").getAttribute('content')` in console.
```
Comment (mindlesscloud) [inline docs/Plugins/gitextractor.md]: To be more specific, this  command should be executed in the **browser console**

Comment (shubham-cmyk) [reply docs/Plugins/gitextractor.md]: Yes in the browser console

**Comment by shubham-cmyk** [comment]
@mindlesscloud  Added Browser keyword

**Pull request merged**